src/GeolocatorService.Reactive/GeolocatorService.Reactive.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>uap10.0.16299;netstandard2.0;xamarinios10;xamarinmac20;MonoAndroid90;monoandroid10.0</TargetFrameworks>
```

Comment:
is it normal that the monoandroid10.0 is not the same casing? It may be ok, just asking :)

Reply:
no clue, I copied this from a Chinook project